src/modules/launcher/PowerLauncher/ResultList.xaml

```diff
@@ -160,13 +161,14 @@
             ScrollViewer.VerticalScrollBarVisibility="Auto"       
             SelectionMode="Single"
             SelectedIndex="{Binding Results.SelectedIndex, Mode=TwoWay}"
-            ItemContainerStyle="{StaticResource ResultsListViewItemContainerStyle}">
+            ItemContainerStyle="{StaticResource ResultsListViewItemContainerStyle}"
+            AutomationProperties.Name="{x:Static props:Resources.Results}">
```

Comment:
[clarification] Shouldn't these resources be dynamic to allow language changes during runtime?

Reply:
I'll make them Dynamic Resources and use the same convention as was done with the rest of the strings in the PowerLauncher project.